server/services/case-service.js

```diff
@@ -36,7 +36,20 @@ const createCaseService = apiUrl => {
       )) || { data: {} }
       return res.data
     },
-    getPagedCaseList: async (
+    getCaseListForBulkMatching: async (
```

Comment:
can be named appropriately. The endpoint should retrieve all the cases with Possible NDelius Record